### PR TITLE
BookreaderUK.GBBook version 4.92b

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103185)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230422170803

- Is there an installer switch to bypass this?

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with: InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230422170803
> 
> * Is there an installer switch to bypass this?

Try common flags

**Comment by upintheairsheep** [comment]
and if that fails, Interactive-Only-Installer

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/b/BookreaderUK/GBBook/4.92b/BookreaderUK.GBBook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BookreaderUK.GBBook
+PackageVersion: 4.92b
+Installers:
+- InstallerUrl: https://www.mqp.com/fun/gbbook.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 288774DF755C3F9E0FB094ECE98EBEEA018DF1DF3BB5F05D9D560A7396AC2CB6
```
Comment (stephengillie) [inline manifests/b/BookreaderUK/GBBook/4.92b/BookreaderUK.GBBook.installer.yaml:10]:
```suggestion
  InstallerSha256: 288774DF755C3F9E0FB094ECE98EBEEA018DF1DF3BB5F05D9D560A7396AC2CB6
  InstallerSwitches:
    Silent: /S
```

File: manifests/b/BookreaderUK/GBBook/4.92b/BookreaderUK.GBBook.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BookreaderUK.GBBook
+PackageVersion: 4.92b
+PackageLocale: en-US
+Publisher: Mariocube
+PackageName: GameBoy Book Reader
```
Comment (stephengillie) [inline manifests/b/BookreaderUK/GBBook/4.92b/BookreaderUK.GBBook.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.mqp.com/fun/
PackageName: GameBoy Book Reader
PackageUrl: http://www.bookreader.co.uk/index.html
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**